src/uuid.cr

```diff
@@ -100,7 +100,7 @@ struct UUID
     end
   end
 
-  # Sets `version`. Doesn't set variant (see `#variant=`).
+  # Sets `version`. Doesn't set variant (see `#variant=`)
```

Comment:
Doc comments SHOULD have dots like every english sentence. Exception messages on the other hand, shouldn't. Please.